modin/conftest.py

```diff
@@ -633,7 +630,11 @@ def s3_base(worker_id, monkeysession):
                     # try again while we still have retries
                     time.sleep(0.1)
             else:
-                raise RuntimeError("Could not connect to moto server after 50 tries.")
+                _, errs = proc.communicate()
```

Comment:
note that this waits for the process to quit... might be needed to `.terminate()` it first to make sure we're not staying in a deadlock?.. but then again, in local scenario the dev might just hit Ctrl-C if stuff is hanging longer than they want to wait...